fastly/resource_fastly_service_acl_entries_v1.go

```diff
@@ -141,14 +141,21 @@ func resourceServiceAclEntriesV1Update(d *schema.ResourceData, meta interface{})
 			ne = new(schema.Set)
 		}
 
-		oes := oe.(*schema.Set)
-		nes := ne.(*schema.Set)
+		oldSet := oe.(*schema.Set)
+		newSet := ne.(*schema.Set)
 
-		removeEntries := oes.Difference(nes).List()
-		addEntries := nes.Difference(oes).List()
+		setDiff := NewSetDiff(func(resource interface{}) (interface{}, error) {
+			// Use the resource ID as the key
+			return resource.(map[string]interface{})["id"], nil
```

Comment:
This is the computed ID within the "entry" object which is what I _think_ is required. 
The reason I selected that as the key is because just before this code is `d.HasChange("entry")`.
@phamann you might be best to sanity check me here.